sql/custom/characters/characters.solocraft.sql

```diff
@@ -1,11 +1,3 @@
-/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
-/*!40101 SET NAMES utf8 */;
-/*!50503 SET NAMES utf8mb4 */;
-/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
-/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
-/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
-
--- Dumping structure for table characters.custom_solocraft_character_stats
 DROP TABLE IF EXISTS `custom_solocraft_character_stats`;
 CREATE TABLE IF NOT EXISTS `custom_solocraft_character_stats` (
   `GUID` tinyint(3) unsigned NOT NULL,
```

Comment:
this is still tinyint though, is that expected ? did you try with a character with guid 1000 ?

Reply:
Oof. My 2nd commit looks like was using the previous formate when i updated the collate.ill fix that in a second.